webpack/ForemanWebhooks/Routes/Webhooks/WebhooksIndexPage/__tests__/WebhooksIndexPage.test.js

```diff
@@ -1,9 +1,10 @@
 import { testComponentSnapshotsWithFixtures } from '@theforeman/test';
-import WebhooksIndexPage from '../WebhooksIndexPage';
+import WebhooksIndexPage from '../../WebhooksIndexPage';
```

Comment:
I think this change causes the following test failure:

![index-test](https://user-images.githubusercontent.com/2664090/108473422-4b595400-728e-11eb-98ab-ef4e075f61ed.png)

You are now importing `WebhooksIndexPage/index.js` which is a connected component and complains about missing store in tests. If you keep the original line, it will import `WebhooksIndexPage/WebhooksIndexPage.js` and that test should no longer crash.

Reply:
well... I thought so too, but after refactoring: `could not find react-redux context value; please ensure the component is wrapped in a <Provider>`

Reply:
I can see it now, it should work when you move redux-related stuff into the parent component.

A couple of things to move around:
1. remove `const dispatch = useDispatch();`  out of `WebhooksIndexPage.js`
2. move `handleCreateSubmit` from `WebhooksdIndexPage.js` to `WebhooksIndexPage/index.js` and bind it to dispatch there, then pass it through props.
3. Call to `useForemanModal` will need to be moved to `index.js` as well, I think you can do something similar like `callWithToastsContext`.

Let me know if you need more details on this.